### PR TITLE
Added ProseLintBear to list of bears

### PR DESCRIPTION
**Pull request by d1ndra**
Fixes [#2284](https://github.com/coala/coala/issues/2284)

### CODE REVIEW

**Comment by AbdealiJK** [comment]
Hello there,

I'm a bit new tot he documentation repo myself. But according to the styleguide in other coala repos the commit message you've given is not right.

 - You shouldn't use past tense and should use imperative tense.
 - You don't have a "tag" in the commit short log
 - There is no "Fixes" keyword in the commit message. You've added it in he PR description, not in the commit message.

Please read the commit guidelines for coala

**Comment by Adrianzatreanu** [comment]
@AbdealiJK is right, these apply to this repo as well, so please respond to his comment :) Thanks!

**Comment by d1ndra** [comment]
@AbdealiJK Made the required changes. Thanks for pointing them out.

**Comment by Adrianzatreanu** [comment]
well aparently the first two commits do the same thing, you should have ammended the first one (using git commit --amend) and it would edit it, but instead, you created another one. you can squash them together while rebasing


you also have a merge commit, we dont want that, could you please rebase correctly and make it go away :p ? if you need help, use ``cobot explain rebase`` on coala.io/chat

File: .coafile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [DOCS]
```
Comment (Nosferatul) [inline .coafile]: Please, add `Closes <link to the issue>` on the commit message.

Comment (d1ndra) [inline .coafile]: Made the changes. @Nosferatul 
@Adrianzatreanu Sorry for the delay.

**Comment by sils** [comment]
@d1ndra did you run coala over the codebase? What are the results?

**Comment by sils** [comment]
commit message and code looks good but I'd rather like to see the output of coala-ci on a pastebin before merging and see if there's any false positives or so

**Comment by d1ndra** [comment]
@sils here the [paste](http://pastebin.com/AUydWyZL)

**Comment by sils** [comment]
hm, seems like two of those are false positives :/ I'm not sure how to continue with this

**Comment by jayvdb** [comment]
I see only one false positive; the prose check of a code block.
If the bear can not skip code blocks in RST, then it does not support RST, and that needs to be created as a task in `coala/coala-bears` and possibly fixed upstream if there is no RST support in the linter (or plugins).
This is another case of code-in-code, and it would be a nice case for when we could remove chunks of a file before passing it to a linter bear which would break on those chunks.

**Comment by sils** [comment]
> More than 30 ppm of exclamations. Keep them under control.

That's about all the punctuation from the rst syntax, not about real punctuation. Maybe we should disable that one?

**Comment by jayvdb** [comment]
[That rule](https://github.com/amperser/proselint/blob/master/proselint/checks/typography/exclamation.py#L45) is only about `!`.

There are only three `!` in that rst file, one is part of a string in a code block, and the other two are literally in the rendered document.

All are optional; IMO the proselint rule is doing what it should be doing here, but we might sneak under that rule if the code block was handled correctly, as that reduces the number of `!` by one.  Apparently based on https://github.com/amperser/proselint/issues/322 and https://github.com/amperser/proselint/issues/542 they are interested in workarounds for false positives like this and generally the code block problem.

**Comment by AbdealiJK** [comment]
Sooo, conclusion here?

**Comment by jayvdb** [comment]
This depends on upstream being fixed, or a different tool being used (https://github.com/coala/documentation/issues/283).  Thank you @d1ndra for helping us better understand the weaknesses of proselint.

**Pull request closed without merging**